software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/couchdb/CouchDBNodeSshDriver.java

```diff
@@ -129,7 +199,7 @@ public void customize() {
     public void launch() {
         log.info("Launching  {}", entity);
         newScript(MutableMap.of(USE_PID_FILE, false), LAUNCHING)
-                .body.append(sudo(String.format("nohup couchdb -p %s -a %s -o couchdb-console.log -e couchdb-error.log -b &", getPidFile(), Os.mergePathsUnix(getRunDir(), getCouchDBConfigFileName()))))
+                .body.append(String.format("nohup ./bin/couchdb -p %s -a %s -o couchdb-console.log -e couchdb-error.log -b &", getPidFile(), Os.mergePathsUnix(getRunDir(), getCouchDBConfigFileName())))
```

Comment:
Thanks for the comment.

Previously CouchDB was installed via binary packages (rpm, deb) that's why the binaries were located in /usr/bin, but the default configuration in the /etc. That's not more the case.
This PR does source installation for the CouchDB. It compiles in the `installDir` and than it copies the files in the `runDir`. `runDir` will be the prefix for all subdirs - bin, share, etc.